python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -162,14 +162,14 @@ def make_request(
     def cancel_job(self, job_id: int):
         self.make_request(endpoint="/jobs/cancel", data={"id": job_id})
 
-    def get_default_workspace(self):
+    def get_default_workspace(self) -> str:
         workspaces = cast(
             List[Dict[str, Any]],
-            check.not_none(self.make_request_cached(endpoint="/workspaces/list", data={})).get(
+            check.not_none(self.make_request_cached(endpoint="/workspaces/list", data={})).get(  # type: ignore  # fmt: skip
                 "workspaces", []
             ),
         )
-        return workspaces[0].get("workspaceId")
+        return workspaces[0]["workspaceId"]
```

Comment:
@benpankow is it safe to assume this is set?

Reply:
The airbyte API docs say yes: https://airbyte-public-api-docs.s3.us-east-2.amazonaws.com/rapidoc-api-docs.html#post-/v1/workspaces/list